### PR TITLE
Update TS and TB InjSept

### PR DESCRIPTION
**Pull request by murilobalves**
- New dipole model based on BD-006 measurements (interpolation of 680A and 720A data, current @ 3GeV is 698.85A
- New optics modes
- New segmented models for septa (with Matrix66 and 6 segments)

### CODE REVIEW

File: pymodels/TS_V04_01/lattice.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+    # --- beam current monitors ---
+    ict    = marker('ICT')
+    fct    = marker('FCT')
+
+    # --- beam position monitors ---
+    bpm    = marker('BPM')
+
+    # --- correctors ---
+    ch     = hcorrector('CH', 0.0)
+    cv     = vcorrector('CV', 0.0)
```
Comment (fernandohds564) [inline pymodels/TS_V04_01/lattice.py]: I think it's better to model the correctors with a thick lens element. This is the way we do it in the other accelerators.
The fact that some of them are inside quadrupoles is not a problem as long as you define their indices in the family_data module

Comment (murilobalves) [reply pymodels/TS_V04_01/lattice.py]: Done

**Comment by xresende** [comment]
@murilobalves , did you do tracking comparisons between _pymodel_ and _AT_ models?

**Comment by murilobalves** [comment]
> @murilobalves , did you do tracking comparisons between _pymodel_ and _AT_ models?

AT and pyaccel linepass tracking match.

**Pull request merged**